<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1345</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #003f5c;

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="table_autos">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>541</width>
      <height>771</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <column>
     <property name="text">
      <string>Model</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Year</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Plate</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="plate">
    <property name="geometry">
     <rect>
      <x>469</x>
      <y>40</y>
      <width>401</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>14</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
font: 14pt &quot;.AppleSystemUIFont&quot;;
border-width:3px;
border-style: solid;
background-color: rgb(255, 255, 255);
border-color: #ffa600;


</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="PlateSearch">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>40</y>
      <width>131</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>14</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-radius:10px;
	font: 14pt &quot;.AppleSystemUIFont&quot;;
border-width:3px;
border-style: solid;
background-color: rgb(255, 255, 255);
border-color: #ffa600;




}
QPushButton:hover{background-color: #bc5090;}



</string>
    </property>
    <property name="text">
     <string>PLATE SEARCH</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>71</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>14</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-radius:10px;
	font: 14pt &quot;.AppleSystemUIFont&quot;;
border-width:3px;
border-style: solid;
background-color: rgb(255, 255, 255);
border-color: #ffa600;




}
QPushButton:hover{background-color: #bc5090;}



</string>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="total">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>181</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>14</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
font: 14pt &quot;.AppleSystemUIFont&quot;;
border-width:3px;
border-style: solid;
background-color: rgb(255, 255, 255);
border-color: #ffa600;


</string>
    </property>
    <property name="text">
     <string>TOTAL:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="quit">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>40</y>
      <width>141</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>14</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-radius:10px;
	font: 14pt &quot;.AppleSystemUIFont&quot;;
border-width:3px;
border-style: solid;
background-color: rgb(255, 255, 255);
border-color: #ffa600;




}
QPushButton:hover{background-color: #bc5090;}



</string>
    </property>
    <property name="text">
     <string>QUIT</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Year_info">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>220</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>14</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: #bc5090;
border-radius:10px;
font: 14pt &quot;.AppleSystemUIFont&quot;;
border-width:2px;
border-style: solid;

</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="City_info">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>220</y>
      <width>231</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>14</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: #bc5090;
border-radius:10px;
font: 14pt &quot;.AppleSystemUIFont&quot;;
border-width:2px;
border-style: solid;

</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="model_info">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>140</y>
      <width>571</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>14</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: #bc5090;
border-radius:10px;
font: 14pt &quot;.AppleSystemUIFont&quot;;
border-width:2px;
border-style: solid;

</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="kilometer_info">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>180</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>14</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: #bc5090;
border-radius:10px;
font: 14pt &quot;.AppleSystemUIFont&quot;;
border-width:2px;
border-style: solid;

</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Price_info">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>180</y>
      <width>231</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>14</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: #bc5090;
border-radius:10px;
font: 14pt &quot;.AppleSystemUIFont&quot;;
border-width:2px;
border-style: solid;

</string>
    </property>
   </widget>
   <widget class="QLabel" name="model">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>140</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>14</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: #bc5090;
border-radius:10px;
font: 14pt &quot;.AppleSystemUIFont&quot;;
border-width:2px;
border-style: solid;

</string>
    </property>
    <property name="text">
     <string>MODEL:</string>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="indent">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="kilometer">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>180</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>14</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: #bc5090;
border-radius:10px;
font: 14pt &quot;.AppleSystemUIFont&quot;;
border-width:2px;
border-style: solid;

</string>
    </property>
    <property name="text">
     <string>KM:</string>
    </property>
    <property name="indent">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="Year">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>220</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>14</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: #bc5090;
border-radius:10px;
font: 14pt &quot;.AppleSystemUIFont&quot;;
border-width:2px;
border-style: solid;

</string>
    </property>
    <property name="text">
     <string>YEAR:</string>
    </property>
    <property name="indent">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="Price">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>180</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>14</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: #bc5090;
border-radius:10px;
font: 14pt &quot;.AppleSystemUIFont&quot;;
border-width:2px;
border-style: solid;

</string>
    </property>
    <property name="text">
     <string>PRICE:</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="indent">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="City1">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>220</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>14</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: #bc5090;
border-radius:10px;
font: 14pt &quot;.AppleSystemUIFont&quot;;
border-width:2px;
border-style: solid;

</string>
    </property>
    <property name="text">
     <string>CITY:</string>
    </property>
    <property name="indent">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="image_label">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>280</y>
      <width>521</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:50px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../elegant.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="city">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>640</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>14</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
	font: 14pt &quot;.AppleSystemUIFont&quot;;
border-width:3px;
border-style: solid;
background-color: rgb(255, 255, 255);
border-color: #ffa600;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>720</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(255, 2, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="image_label_2">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>620</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../elegant.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>690</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(255, 2, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="image_label_3">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>620</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../elegant.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>720</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(255, 2, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>690</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(255, 2, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="image_label_4">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>620</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../elegant.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>720</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(255, 2, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>690</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(255, 2, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
